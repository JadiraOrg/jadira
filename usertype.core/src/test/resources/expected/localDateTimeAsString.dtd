<!ELEMENT dataset (
    LOCALDATETIMEASSTRING*)>

<!ELEMENT LOCALDATETIMEASSTRING EMPTY>
<!ATTLIST LOCALDATETIMEASSTRING
    ID CDATA #REQUIRED
    LOCALDATETIME CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

