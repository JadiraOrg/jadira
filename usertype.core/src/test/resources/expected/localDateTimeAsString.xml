<?xml version='1.0' encoding='UTF-8'?>
<dataset>
  <localDateTimeAsString ID="0" LOCALDATETIME="2004-02-25T12:11:10.0" NAME="test_0"/>
  <localDateTimeAsString ID="1" LOCALDATETIME="1980-03-11T13:12:11.0" NAME="test_1"/>
</dataset>
